pkg/i2gw/providers/gce/ir_converter_test.go

```diff
@@ -883,6 +886,119 @@ func Test_convertToIR(t *testing.T) {
 			},
 			expectedErrors: field.ErrorList{},
 		},
+		{
+			name: "ingress with a Frontend Config specifying Ssl Policy",
+			ingresses: map[types.NamespacedName]*networkingv1.Ingress{
```

Comment:
These kinds of tests can become really difficult to follow if you have to spell out the entirety of each manifest for each test case. I'd recommend moving towards a pattern where there's a base resource that each test case makes small modifications to. Here's an example of that pattern in Kubernetes: https://github.com/kubernetes/kubernetes/blob/80941e3e87d3909aa70e4f38f37b447f4376661f/pkg/apis/storage/validation/validation_test.go#L1741.

Reply:
Sounds good! Just updated based on the example.
For now I only make sure in each test case, we modify the input ingress based on a template ingress, and I'm still keeping the expected output in details to make sure we compare with to detailed outputs.